README.md

```diff
@@ -25,15 +26,17 @@ Lots of results for the new experiment.
 4. KDEX is part of the "Space Exploration" tech node.
 
 ## License
-This mod is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) license; effectively it means it's free to use and modify for non-commercial usage as long as the original author(s) are attributed.
-
+This mod is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) license; effectively it means it's free to use and modify for non-commercial usage as long as the original author(s) are attributed.  
+![CC BY-NC-SA 3.0](https://licensebuttons.net/l/by-nc-sa/3.0/88x31.png)  
 ## Resources
 * Source code
   + [GitHub](http://github.com/mwerle/kdex)
 * Forum threads
   + [Micha's Release Thread](http://forum.kerbalspaceprogram.com/index.php?showtopic=166619)
   + [masTerTorch's original Thread](http://forum.kerbalspaceprogram.com/index.php?showtopic=50039)
 * Download Links
-  + [GitHub](http://github.com/mwerle/kdex/releases)
+  + [GitHub](http://github.com/mwerle/kdex/releases/latest)
   + [SpaceDock](http://spacedock.info/mod/1551)
   + [CurseForge](https://www.curseforge.com/kerbal/ksp-mods/kdex-continued)
+  
+<!-- zed'K was here. ;p -->
```

Comment:
Best to add a "Contributors" section and put your name up in lights ;)

Reply:
And/or add a Contributors.md file and link it here.

Reply:
am honored. thank you.

just hide a "zed'K was here" somewhere - like in the sun. 📦

Reply:
[zed'K was here](https://forum.kerbalspaceprogram.com/index.php?/profile/190933-zer0kerbal/)